packages/react-cosmos-core/src/fixtureTree/flattenFixtureTree.tsx

```diff
@@ -22,7 +22,10 @@ export function flattenFixtureTree(
       const childNode = children[childName];
       const { data: childData } = childNode;
 
-      if (childData.type === 'fileDir')
+      if (
+        childData.type === 'fileDir' ||
+        (childData.type === 'fixture' && childName === 'fixture')
```

Comment:
@ThaisRobba Is this change needed? The tests seem to pass without it and I think once the fixture tree was created the fixture names no longer matter when flattening it.

Reply:
@skidding You are totally correct - not sure why it wasn't working until then for me (I probably forgot to rebuild at some point). I've reverted this change ^^